src/main/java/seedu/loyaltylift/model/order/StatusValue.java

```diff
@@ -35,6 +35,18 @@ public StatusValue nextValue() {
         return StatusValue.values()[pos + 1];
     }
 
+    /**
+     * Returns a {@code StatusValue}, "Cancelled"
+     * Should not be called when StatusValue is Completed.
+     * @return A {@code StatusValue}
+     */
+    public StatusValue changeToCancelled() {
```

Comment:
Same as the comment in `StatusUpdate.java`, does not feel necessary to have this abstraction